charm/charmcraft.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: repo-policy-compliance
+
+type: charm
+
+bases:
+  - build-on:
+    - name: ubuntu
+      channel: "22.04"
+    run-on:
+    - name: ubuntu
+      channel: "22.04"
+
+summary: Check the repository setup for policy compliance
+
+description: |
+  Used to check whether a GitHub repository complies with expected policies.
+
+extensions:
+  - flask-framework
+
+requires:
+  postgresql:
+    interface: postgresql_client
+    limit: 1
+
+config:
+  options:
+    charm_token:
+      description: The token for the charm to generate new tokens.
```

Comment:
Would you mind helping specify where this token can be generated from? Same for github_token perhaps?
I think that will let other deployments have easier time figuring out how to use this charm. A lot of teams seem to have trouble with the compliance policies :D